src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/FieldsSections.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC, useState } from 'react';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { ExpandableSection } from '@patternfly/react-core';
+
+import { FieldGroup } from './FieldGroup';
+import { useDrawerContext } from './useDrawerContext';
+import { setTemplateParameterValue } from './utils';
+
+const FieldsSections: FC = () => {
+  const { t } = useKubevirtTranslation();
+
+  const { setTemplate, template } = useDrawerContext();
+  const requiredParameters = template?.parameters?.filter(
+    (param) => param.required && param.name !== 'NAME',
+  );
+  const optionalParameters = template?.parameters?.filter(
+    (param) => !param.required && param.name !== 'NAME',
+  );
+  const [requiredSectionOpen, setRequiredSectionOpen] = useState(requiredParameters.length > 0);
```

Comment:
If the template has requiredParameters, expand the section as we need user attention on that.
If there are no required parameters, don't display that section